psalm-baseline.xml

```diff
@@ -53,6 +53,25 @@
     <PossiblyNullPropertyAssignmentValue occurrences="1">
       <code>null</code>
     </PossiblyNullPropertyAssignmentValue>
+    <UndefinedClass occurrences="2">
```

Comment:
Not sure where these come from... they definitely weren't in the psalm errors before. Maybe difference local run vs. travis environment?

Reply:
Maybe because I don't have Redis installed? Pretty stupid check, then… :)

But no… I have ext-redis installed. 🤷‍♂️